src/lib/utils/SynapseConstants.ts

```diff
@@ -130,3 +130,6 @@ export const BASE = 1024,
 
 export const NETWORK_UNAVAILABLE_MESSAGE =
   'This site cannot be reached. Either a connection is unavailable, or your network administrator has blocked you from accessing this site.'
+
+/* The ID of the table in production that contains the canned responses for the submission rejection form. */
+export const REJECT_SUBMISSION_CANNED_RESPONSES_TABLE = 'syn50683097'
```

Comment:
This is currently in my personal test project but I'm working on moving the table to an ACT project where it can be more easily discovered and edited.